frontend/public/locales/fr/translation.json

```diff
@@ -311,6 +311,16 @@
     "exportAllData": "Télécharger vos données",
     "deleteAccount": "Supprimer votre compte"
   },
+  "pollUserSettingsForm": {
+    "errorOccurredDuringPreferencesUpdate": "Désolé, une erreur est survenue, impossible de mettre à jour vos préférences.",
+    "preferencesUpdatedSuccessfully": "Préférences mises à jour avec succès.",
+    "rateLater": "Liste à comparer plus tard",
+    "autoRemove": "Suppression automatique",
+    "autoRemoveHelpText_one": "Les entités seront supprimées de votre liste à comparer plus tard après {{rateLaterAutoRemove}} comparison.",
+    "autoRemoveHelpText_many": "Les entités seront supprimées de votre liste à comparer plus tard après {{rateLaterAutoRemove}} comparisons.",
+    "autoRemoveHelpText_other": "Les entités seront supprimées de votre liste à comparer plus tard après {{rateLaterAutoRemove}} comparisons.",
```

Comment:
maybe "élements" instead of "entités"?

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1501/commits/ab1b0ead398c986fab8854d74271706eab1df673